<!DOCTYPE article PUBLIC "-//NLM//DTD Journal Archiving and Interchange DTD v2.3 20070202//EN" "archivearticle.dtd">
<article xmlns:xlink="http://www.w3.org/1999/xlink" article-type="review-article">
	<?properties open_access?>
	<front>
		<journal-meta>
			<journal-id journal-id-type="nlm-ta">Genome Biol</journal-id>
			<journal-title>Genome Biology</journal-title>
			<issn pub-type="ppub">1465-6906</issn>
			<issn pub-type="epub">1465-6914</issn>
			<publisher>
				<publisher-name>BioMed Central</publisher-name>
				<publisher-loc>London</publisher-loc>
			</publisher>
		</journal-meta>
		<article-meta>
			<article-id pub-id-type="pmid">12537580</article-id>
			<article-id pub-id-type="pmc">151193</article-id>
			<article-id pub-id-type="publisher-id">gb-2002-3-12-reviews1035</article-id>
			<article-categories>
				<subj-group subj-group-type="heading">
					<subject>Minireview</subject>
				</subj-group>
			</article-categories>
			<title-group>
				<article-title>RNAi hushes heterochromatin</article-title>
			</title-group>
			<contrib-group>
				<contrib id="A1" contrib-type="author">
					<name>
						<surname>Bailis</surname>
						<given-names>Julie M</given-names>
					</name>
					<xref ref-type="aff" rid="I1">1</xref>
				</contrib>
				<contrib id="A2" corresp="yes" contrib-type="author">
					<name>
						<surname>Forsburg</surname>
						<given-names>Susan L</given-names>
					</name>
					<xref ref-type="aff" rid="I1">1</xref>
					<email>forsburg@salk.edu</email>
				</contrib>
			</contrib-group>
			<aff id="I1">
				<label>1</label>Molecular and Cellular Biology Laboratory, The Salk Institute, 10010 North Torrey Pines Road, La Jolla, CA 92037, USA</aff>
			<aff>Correspondence: Susan L Forsburg. E-mail: forsburg@salk.edu</aff>
			<pub-date pub-type="ppub">
				<year>2002</year>
			</pub-date>
			<pub-date pub-type="epub">
				<day>28</day>
				<month>11</month>
				<year>2002</year>
			</pub-date>
			<volume>3</volume>
			<issue>12</issue>
			<fpage>reviews1035.1</fpage>
			<lpage>reviews1035.4</lpage>
			<ext-link ext-link-type="uri" xlink:href="http://genomebiology.com/2002/3/12/reviews/1035"/>
			<copyright-statement>Copyright &#x000a9; 2002 BioMed Central Ltd</copyright-statement>
			<abstract abstract-type="short">
				<p>Recent work in the fission yeast <italic>Schizosaccharomyces pombe </italic>reveals that RNA is an integral component of silent heterochromatin.</p>
			</abstract>
			<abstract>
				<p>Repeated DNA elements and region-specific protein modifications combine within chromosomes to form a transcriptionally silent chromatin structure called heterochromatin. Recent work in the fission yeast <italic>Schizosaccharomyces pombe </italic>reveals that RNA is also an integral component of silent heterochromatin, providing a new perspective on how heterochromatin is organized and maintained in eukaryotic cells.</p>
			</abstract>
		</article-meta>
	</front>
	<body>
		<p>The heterochromatin of eukaryotic genomes is concentrated at and around centromeres and plays a central role in the maintenance of genome integrity through regulation of both gene expression and chromosome segregation [<xref ref-type="bibr" rid="B1">1</xref>,<xref ref-type="bibr" rid="B2">2</xref>]. Centric heterochromatin is required for cohesion between sister centromeres, which ensures proper chromosome segregation at mitosis [<xref ref-type="bibr" rid="B3">3</xref>,<xref ref-type="bibr" rid="B4">4</xref>]. Additional heterochromatin at telomeres protects the DNA at chromosome ends from inappropriate fusions and recombination. If genes are inserted into heterochromatin they are transcriptionally silenced; this silencing is maintained epigenetically (independent of DNA sequence) and can be propagated through multiple cell divisions. Interestingly, transposable elements and repeated DNA arrays are abundant within heterochromatin [<xref ref-type="bibr" rid="B1">1</xref>,<xref ref-type="bibr" rid="B5">5</xref>], suggesting that the molecular nature of repeated DNA sequences is related to assembly of this specialized chromatin structure. Two recent studies have implicated a new and unexpected player in heterochromatin assembly: RNA interference (RNAi; [<xref ref-type="bibr" rid="B6">6</xref>,<xref ref-type="bibr" rid="B7">7</xref>]).</p>
		<p>The specification of a region for heterochromatin assembly results from a 'histone code' that modifies the tails of histone proteins within nucleosomes in heterochromatin differentially from those in euchromatin [<xref ref-type="bibr" rid="B8">8</xref>,<xref ref-type="bibr" rid="B9">9</xref>]. The fission yeast <italic>Schizosaccharomyces pombe </italic>provides an important model for heterochromatin assembly in eukaryotic cells, because the regions of heterochromatin have been molecularly defined (the mating-type locus, telomeres, and inverted repeats surrounding centromeres), and the evolutionary conserved proteins required for heterochromatin formation have been characterized. Several steps in the pathway of heterochromatin assembly have been identified in <italic>S. pombe</italic>. The histone deacetylase (HDAC) Clr3 deacetylates one specific residue, lysine 9 of histone H3; lysine 14 of histone H3 is deacetylated by other HDACs, potentially Clr6 and/or Hda1 [<xref ref-type="bibr" rid="B10">10</xref>]. Deacetylation of histone H3 lysine 9 allows this residue to be methylated by the Clr4 methyltransferase (the homolog of <italic>Drosophila </italic>Su(var)3-9), which acts together with Rik1, a protein that contains WD40 repeats, domains predicted to be involved in protein-protein interactions [<xref ref-type="bibr" rid="B10">10</xref>,<xref ref-type="bibr" rid="B11">11</xref>]. The heterochromatin protein 1 (HP1) homolog Swi6 then binds to the methylated lysine 9 of histone H3 [<xref ref-type="bibr" rid="B10">10</xref>] and spreads the silent heterochromatin structure (with methylated lysine 9) to surrounding sequences [<xref ref-type="bibr" rid="B12">12</xref>]. Spreading is ultimately blocked by boundary elements that define the extent of the heterochromatic region [<xref ref-type="bibr" rid="B11">11</xref>,<xref ref-type="bibr" rid="B12">12</xref>,<xref ref-type="bibr" rid="B13">13</xref>]. Interestingly, both the Clr4 methyltransferase and Swi6 contain chromodomains, protein motifs that are capable of binding RNA as well as methylated histone tails [<xref ref-type="bibr" rid="B14">14</xref>,<xref ref-type="bibr" rid="B15">15</xref>]. Once established, transcriptionally silent heterochromatin can be passed to successive generations of cells epigenetically through mitosis and meiosis [<xref ref-type="bibr" rid="B16">16</xref>].</p>
		<p>Two recent studies have revealed a surprising involvement for RNAi in heterochromatin assembly [<xref ref-type="bibr" rid="B6">6</xref>,<xref ref-type="bibr" rid="B7">7</xref>]. RNAi was first identified as a mechanism of gene silencing in <italic>Caenorhabditis elegans </italic>and other animals; it also acts in the related processes of quelling in <italic>Neurospora </italic>and post-transcriptional gene silencing in plants [<xref ref-type="bibr" rid="B17">17</xref>]. RNAi involves a conserved group of proteins: Dicer, Argonaute, and RNA-dependent RNA polymerase (RdRP) [<xref ref-type="bibr" rid="B17">17</xref>,<xref ref-type="bibr" rid="B18">18</xref>]. Double-stranded RNA (dsRNA) amplified by RdRP is processed by the Dicer enzyme to form small interfering RNAs (siRNAs) of 21-25 nucleotides, which bind to an RNA-induced silencing complex (RISC) containing Argonaute. The siRNAs then target the mRNA homologous to the siRNA for destruction. The RISC complex is thought to promote methylation of the DNA [<xref ref-type="bibr" rid="B17">17</xref>,<xref ref-type="bibr" rid="B18">18</xref>], which further inhibits transcription in the region.</p>
		<p>The two recent studies [<xref ref-type="bibr" rid="B6">6</xref>,<xref ref-type="bibr" rid="B7">7</xref>] implicate these same RNAi enzymes specifically in the initiation of chromatin silencing and heterochromatin assembly in <italic>S. pombe</italic>. These findings additionally indicate that maintenance of silencing requires other mechanisms, including Swi6-dependent spreading of heterochromatin. This allows the definition of a pathway of heterochromatin assembly that differentiates between factors required for the establishment and the propagation of a heterochromatin domain (Figure <xref ref-type="fig" rid="F1">1</xref>). Homologs of the genes encoding Dicer (<italic>dcr1</italic>
			<sup>+</sup>), Argonaute (<italic>ago1</italic>
			<sup>+</sup>) and RdRP (<italic>rdp1</italic>
			<sup>+</sup>), identified by the fission yeast genome project [<xref ref-type="bibr" rid="B19">19</xref>], were cloned and disrupted. The mutants are viable but defective in chromosome transmission and centromeric silencing. A mechanism underlying the role for RNAi in these processes was suggested by the accumulation of RNA transcripts in the mutants that correspond to heterochromatic centromere repeat sequences. This implies that processing of RNAs derived from these sequences is required to silence centromeric heterochromatin. Consistent with this view, the RdRP protein associates with the centromeric repeats <italic>in vivo</italic>, raising the possibility that dsRNA arises from RdRP activity on a single transcribed strand of RNA [<xref ref-type="bibr" rid="B6">6</xref>]. Repeated rounds of this process would lead to amplification of the RNAs, which presumably target other factors to the heterochromatin to initiate silencing.</p>
		<p>Curiously, few RNA transcripts from the centromere were observed in wild-type cells [<xref ref-type="bibr" rid="B6">6</xref>]. But in an independent study, a search for potential Dicer cleavage products in <italic>S. pombe </italic>revealed sequences homologous to the centromeric repeats even in wild-type cells [<xref ref-type="bibr" rid="B20">20</xref>], suggesting that the mechanism is finely tuned and requires relatively little RNA. Because sequences homologous to telomeres and the mating-type locus were not revealed by these studies, it will be important to identify the RNA transcripts that act at these non-centric sites of heterochromatin.</p>
		<p>To determine at what stage of the pathway RNAi components act, Volpe and colleagues [<xref ref-type="bibr" rid="B6">6</xref>] used chromatin immunoprecipitation to demonstrate that RNAi is required for efficient methylation of lysine 9 of histone H3. An obvious model is that siRNAs or the RNAi protein components directly target the appropriate enzymes to histones, promoting deacetylation and methylation. Indirect support for this model is provided by the observation that chromodomain proteins such as Clr4 and Swi6 can bind both methylated histone tails and RNA [<xref ref-type="bibr" rid="B14">14</xref>,<xref ref-type="bibr" rid="B15">15</xref>]. Direct evidence that the dsRNA or siRNA binds these proteins remains to be determined.</p>
		<p>Interestingly, assembly of the silent heterochromatin at the mating-type locus requires a sequence, <italic>cenH</italic>, that is related to the centromere repeats [<xref ref-type="bibr" rid="B7">7</xref>,<xref ref-type="bibr" rid="B21">21</xref>]. A region containing this sequence is sufficient to promote silencing at an ectopic site in an RNAi-dependent fashion [<xref ref-type="bibr" rid="B7">7</xref>]. Because epigenetic inheritance of the silent heterochromatin at the <italic>mat </italic>(mating-type) locus is extremely robust, Hall and colleagues [<xref ref-type="bibr" rid="B7">7</xref>] used the <italic>cenH </italic>site at the <italic>mat </italic>locus to further analyze the role of RNAi proteins in the establishment of silent heterochromatin. The histone methylation required for the assembly of silent heterochromatin can be blocked by inhibiting the upstream histone deacetylase with Trichostatin A or by deleting the Clr4 methyltransferase itself. Both of these treatments erase the epigenetic mark for silencing [<xref ref-type="bibr" rid="B22">22</xref>,<xref ref-type="bibr" rid="B23">23</xref>]. Significantly, in the RNAi mutants, methylation and silencing at the mating-type locus could not be re-established when Trichostatin A was removed or Clr4 was restored [<xref ref-type="bibr" rid="B7">7</xref>]. In striking contrast, a previously established, epigenetically inherited silent chromatin state could be maintained in the absence of RNAi proteins, suggesting that RNAi is required specifically for heterochromatin assembly but is dispensable for heterochromatin maintenance. This suggests a linear pathway from RNAi activity to histone H3 lysine 9 methylation to Swi6 binding, consistent with prior observations that Swi6 association with heterochromatin requires prior methylation of histone H3 lysine 9 [<xref ref-type="bibr" rid="B10">10</xref>,<xref ref-type="bibr" rid="B11">11</xref>].</p>
		<p>Unexpectedly, although Swi6 appears to act downstream of histone methylation, Hall and colleagues [<xref ref-type="bibr" rid="B7">7</xref>] found that Swi6 itself is required to maintain the methylated state and to promote spreading of heterochromatin. In a <italic>swi6 </italic>mutant, methylation of histone H3 lysine 9 within the mating-type locus is restricted to <italic>cenH </italic>rather than distributed throughout a broad region as in wild-type cells. This indicates that binding of Swi6 to methylated histone H3 lysine 9 at <italic>cenH </italic>is required for further methylation outside this region, resulting in a positive feedback mechanism that spreads silent chromatin. This observation is consistent with the ability of <italic>cenH </italic>to nucleate silencing at an ectopic chromosomal site, and defines two steps in heterochromatin assembly: the initial methylation event triggered by RNAi, and the Swi6-mediated spreading of the silent chromatin state toward the boundaries with euchromatin.</p>
		<p>This result is consistent with a similar study by Partridge and colleagues [<xref ref-type="bibr" rid="B24">24</xref>]. Using silencing at an ectopic site as an assay, they define a sequence from the centromere repeats - related to <italic>cenH </italic>- that is sufficient to promote histone H3 lysine 9 methylation and silencing. Histone methylation of this site itself is independent of Swi6, similar to the situation for <italic>cenH</italic>. But methylation at the centromere sequence requires another chromodomain protein, Chp1 [<xref ref-type="bibr" rid="B24">24</xref>]. Thus, there appear to be different requirements for the two chromodomain proteins Chp1 and Swi6 during heterochromatin nucleation, although both proteins subsequently display a similar localization pattern across the centromere [<xref ref-type="bibr" rid="B12">12</xref>].</p>
		<p>These new studies [<xref ref-type="bibr" rid="B6">6</xref>,<xref ref-type="bibr" rid="B7">7</xref>], as well as recent work from a number of other laboratories, have uncovered important details about the formation and function of heterochromatin in fission yeast. Although heterochromatin commonly contains repeated DNA arrays, this work demonstrates that RNA components also act in coordination with region-specific histone modification to drive the association of heterochromatin-specific factors and spreading of a silent chromatin structure. Notably, this work emphasizes the mechanistic similarities in diverse cellular processes. In plants and <italic>C. elegans</italic>, RNAi is known to function in gene silencing, probably by promoting DNA methylation [<xref ref-type="bibr" rid="B17">17</xref>]. Maintenance of male fertility in <italic>Drosophila </italic>requires the RNAi-like silencing of germline transcripts [<xref ref-type="bibr" rid="B25">25</xref>]. In <italic>Tetrahymena</italic>, a strong link has been uncovered between heterochromatin formation and programmed DNA elimination during cell growth [<xref ref-type="bibr" rid="B26">26</xref>]. During X-chromosome inactivation in mammals, methylation of histone H3 lysine 9 at a specific site targets an RNA, <italic>Xist</italic>, to spread its silent epigenetic mark along the length of the chromosome [<xref ref-type="bibr" rid="B27">27</xref>]. Furthermore, a pericentromeric RNA component has recently been shown to be required for histone methylation and heterochromatin formation in mouse [<xref ref-type="bibr" rid="B28">28</xref>]. It is also important to note, however, that in at least one eukaryote, <italic>Saccharomyces cerevisiae</italic>, there are no apparent homologs of the RNAi machinery, Swi6/HP1 proteins or methyltransferases; transcriptional silencing is achieved through a pathway that employs different factors, the 'silent information regulator' (SIR) proteins [<xref ref-type="bibr" rid="B29">29</xref>].</p>
		<p>The identification of RNAi as a core component of the chromatin silencing mechanism increases the known complexity of formation of higher-order chromatin structure. Many details of this process, such as the initiation of RNA transcription from specific heterochromatin sequences and the mechanism by which methyltransferases are targeted to heterochromatin by siRNAs, remain to be determined. In addition, it is not known how histone H3 lysine 9 methylation and Swi6 cooperate to promote spreading of the silent chromatin state or whether this process involves other chromosomal factors. And there are likely to be subtle differences in the pathways that establish heterochromatin at different sites in the genome. While Chp1 is required to silence centromere sequences, it does not appear to affect silencing at the mating-type locus (see [<xref ref-type="bibr" rid="B24">24</xref>] and references therein). In addition, although RNAi mutants lose silencing at centromeres, the mutants do not display obvious defects in mating or in mating-type silencing, suggesting that the <italic>mat </italic>locus is better able to maintain its epigenetic imprint [<xref ref-type="bibr" rid="B6">6</xref>,<xref ref-type="bibr" rid="B7">7</xref>]. This strongly suggests that there will prove to be region-specific modifiers to the common pathway of heterochromatin assembly. Although many of the molecular components of the pathway are now defined, and linked mechanistically to other cellular processes, it is clear that there are still tantalizing puzzles to solve in the silence of heterochromatin.</p>
	</body>
	<back>
		<ack>
			<sec>
				<title>Acknowledgements</title>
				<p>Work in S.L.F.'s laboratory is supported by the NIH and the American Cancer Society. S.L.F. is a Stohlman Scholar of the Leukemia and Lymphoma Society. J.M.B. is a fellow of the Damon Runyon Cancer Research Foundation.</p>
			</sec>
		</ack>
		<ref-list>
			<ref id="B1">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Henikoff</surname>
							<given-names>S</given-names>
						</name>
					</person-group>
					<article-title>Heterochromatin function in complex genomes.</article-title>
					<source>Biochem Biophys Acta</source>
					<year>2000</year>
					<volume>1470</volume>
					<fpage>1</fpage>
					<lpage>8</lpage>
				</citation>
			</ref>
			<ref id="B2">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Dillon</surname>
							<given-names>N</given-names>
						</name>
						<name>
							<surname>Festenstein</surname>
							<given-names>R</given-names>
						</name>
					</person-group>
					<article-title>Unravelling heterochromatin: competition between positive and negative factors regulates accessibility.</article-title>
					<source>Trends Genet</source>
					<year>2002</year>
					<volume>18</volume>
					<fpage>252</fpage>
					<lpage>258</lpage>
					<pub-id pub-id-type="pmid">12047950</pub-id>
					<pub-id pub-id-type="doi">10.1016/S0168-9525(02)02648-3</pub-id>
				</citation>
			</ref>
			<ref id="B3">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Bernard</surname>
							<given-names>P</given-names>
						</name>
						<name>
							<surname>Maure</surname>
							<given-names>JF</given-names>
						</name>
						<name>
							<surname>Partridge</surname>
							<given-names>JF</given-names>
						</name>
						<name>
							<surname>Genier</surname>
							<given-names>S</given-names>
						</name>
						<name>
							<surname>Javerzat</surname>
							<given-names>JP</given-names>
						</name>
						<name>
							<surname>Allshire</surname>
							<given-names>RC</given-names>
						</name>
					</person-group>
					<article-title>Requirement of heterochromatin for cohesion at centromeres.</article-title>
					<source>Science</source>
					<year>2001</year>
					<volume>294</volume>
					<fpage>2539</fpage>
					<lpage>2542</lpage>
					<pub-id pub-id-type="pmid">11598266</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1064027</pub-id>
				</citation>
			</ref>
			<ref id="B4">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Nonaka</surname>
							<given-names>N</given-names>
						</name>
						<name>
							<surname>Kitajima</surname>
							<given-names>T</given-names>
						</name>
						<name>
							<surname>Yokobayashi</surname>
							<given-names>S</given-names>
						</name>
						<name>
							<surname>Xiao</surname>
							<given-names>G</given-names>
						</name>
						<name>
							<surname>Yamamoto</surname>
							<given-names>M</given-names>
						</name>
						<name>
							<surname>Grewal</surname>
							<given-names>SI</given-names>
						</name>
						<name>
							<surname>Watanabe</surname>
							<given-names>Y</given-names>
						</name>
					</person-group>
					<article-title>Recruitment of cohesin to heterochromatic regions by Swi6/HP1 in fission yeast.</article-title>
					<source>Nat Cell Biol</source>
					<year>2002</year>
					<volume>4</volume>
					<fpage>89</fpage>
					<lpage>93</lpage>
					<pub-id pub-id-type="pmid">11780129</pub-id>
					<pub-id pub-id-type="doi">10.1038/ncb739</pub-id>
				</citation>
			</ref>
			<ref id="B5">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Avramova</surname>
							<given-names>ZV</given-names>
						</name>
					</person-group>
					<article-title>Heterochromatin in animals and plants. Similarities and differences.</article-title>
					<source>Plant Physiol</source>
					<year>2002</year>
					<volume>129</volume>
					<fpage>40</fpage>
					<lpage>49</lpage>
					<pub-id pub-id-type="pmid">12011336</pub-id>
					<pub-id pub-id-type="doi">10.1104/pp.010981</pub-id>
				</citation>
			</ref>
			<ref id="B6">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Volpe</surname>
							<given-names>TA</given-names>
						</name>
						<name>
							<surname>Kidner</surname>
							<given-names>C</given-names>
						</name>
						<name>
							<surname>Hall</surname>
							<given-names>IM</given-names>
						</name>
						<name>
							<surname>Teng</surname>
							<given-names>G</given-names>
						</name>
						<name>
							<surname>Grewal</surname>
							<given-names>SI</given-names>
						</name>
						<name>
							<surname>Martienssen</surname>
							<given-names>RA</given-names>
						</name>
					</person-group>
					<article-title>Regulation of heterochromatic silencing and histone H3 lysine-9 methylation by RNAi.</article-title>
					<source>Science</source>
					<year>2002</year>
					<volume>297</volume>
					<fpage>1833</fpage>
					<lpage>1837</lpage>
					<pub-id pub-id-type="pmid">12193640</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1074973</pub-id>
				</citation>
			</ref>
			<ref id="B7">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Hall</surname>
							<given-names>IM</given-names>
						</name>
						<name>
							<surname>Shankaranarayana</surname>
							<given-names>GD</given-names>
						</name>
						<name>
							<surname>Noma</surname>
							<given-names>K</given-names>
						</name>
						<name>
							<surname>Ayoub</surname>
							<given-names>N</given-names>
						</name>
						<name>
							<surname>Cohen</surname>
							<given-names>A</given-names>
						</name>
						<name>
							<surname>Grewal</surname>
							<given-names>SI</given-names>
						</name>
					</person-group>
					<article-title>Establishment and maintenance of a heterochromatin domain.</article-title>
					<source>Science</source>
					<year>2002</year>
					<volume>297</volume>
					<fpage>2232</fpage>
					<lpage>2237</lpage>
					<pub-id pub-id-type="pmid">12215653</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1076466</pub-id>
				</citation>
			</ref>
			<ref id="B8">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Strahl</surname>
							<given-names>BD</given-names>
						</name>
						<name>
							<surname>Allis</surname>
							<given-names>CD</given-names>
						</name>
					</person-group>
					<article-title>The language of covalent histone modifications.</article-title>
					<source>Nature</source>
					<year>2000</year>
					<volume>403</volume>
					<fpage>41</fpage>
					<lpage>45</lpage>
					<pub-id pub-id-type="pmid">10638745</pub-id>
					<pub-id pub-id-type="doi">10.1038/47412</pub-id>
				</citation>
			</ref>
			<ref id="B9">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Jenuwein</surname>
							<given-names>T</given-names>
						</name>
						<name>
							<surname>Allis</surname>
							<given-names>CD</given-names>
						</name>
					</person-group>
					<article-title>Translating the histone code.</article-title>
					<source>Science</source>
					<year>2001</year>
					<volume>293</volume>
					<fpage>1074</fpage>
					<lpage>1080</lpage>
					<pub-id pub-id-type="pmid">11498575</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1063127</pub-id>
				</citation>
			</ref>
			<ref id="B10">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Nakayama</surname>
							<given-names>J</given-names>
						</name>
						<name>
							<surname>Rice</surname>
							<given-names>JC</given-names>
						</name>
						<name>
							<surname>Strahl</surname>
							<given-names>BD</given-names>
						</name>
						<name>
							<surname>Allis</surname>
							<given-names>CD</given-names>
						</name>
						<name>
							<surname>Grewal</surname>
							<given-names>SI</given-names>
						</name>
					</person-group>
					<article-title>Role of histone H3 lysine 9 methylation in epigenetic control of heterochromatin assembly.</article-title>
					<source>Science</source>
					<year>2001</year>
					<volume>292</volume>
					<fpage>110</fpage>
					<lpage>113</lpage>
					<pub-id pub-id-type="pmid">11283354</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1060118</pub-id>
				</citation>
			</ref>
			<ref id="B11">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Noma</surname>
							<given-names>K</given-names>
						</name>
						<name>
							<surname>Allis</surname>
							<given-names>CD</given-names>
						</name>
						<name>
							<surname>Grewal</surname>
							<given-names>SI</given-names>
						</name>
					</person-group>
					<article-title>Transitions in distinct histone H3 methylation patterns at the heterochromatin domain boundaries.</article-title>
					<source>Science</source>
					<year>2001</year>
					<volume>293</volume>
					<fpage>1150</fpage>
					<lpage>1155</lpage>
					<pub-id pub-id-type="pmid">11498594</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1064150</pub-id>
				</citation>
			</ref>
			<ref id="B12">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Partridge</surname>
							<given-names>JF</given-names>
						</name>
						<name>
							<surname>Borgstrom</surname>
							<given-names>B</given-names>
						</name>
						<name>
							<surname>Allshire</surname>
							<given-names>RC</given-names>
						</name>
					</person-group>
					<article-title>Distinct protein interaction domains and protein spreading in a complex centromere.</article-title>
					<source>Genes Dev</source>
					<year>2000</year>
					<volume>14</volume>
					<fpage>783</fpage>
					<lpage>791</lpage>
					<pub-id pub-id-type="pmid">10766735</pub-id>
				</citation>
			</ref>
			<ref id="B13">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Thon</surname>
							<given-names>G</given-names>
						</name>
						<name>
							<surname>Bjerling</surname>
							<given-names>P</given-names>
						</name>
						<name>
							<surname>Bunner</surname>
							<given-names>CM</given-names>
						</name>
						<name>
							<surname>Verhein-Hansen</surname>
							<given-names>J</given-names>
						</name>
					</person-group>
					<article-title>Expression-state boundaries in the mating-type region of fission yeast.</article-title>
					<source>Genetics</source>
					<year>2002</year>
					<volume>161</volume>
					<fpage>611</fpage>
					<lpage>622</lpage>
					<pub-id pub-id-type="pmid">12072458</pub-id>
				</citation>
			</ref>
			<ref id="B14">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Akhtar</surname>
							<given-names>A</given-names>
						</name>
						<name>
							<surname>Zink</surname>
							<given-names>D</given-names>
						</name>
						<name>
							<surname>Becker</surname>
							<given-names>PB</given-names>
						</name>
					</person-group>
					<article-title>Chromodomains are protein-RNA interaction modules.</article-title>
					<source>Nature</source>
					<year>2000</year>
					<volume>407</volume>
					<fpage>405</fpage>
					<lpage>409</lpage>
					<pub-id pub-id-type="pmid">11014199</pub-id>
					<pub-id pub-id-type="doi">10.1038/35030169</pub-id>
				</citation>
			</ref>
			<ref id="B15">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Muchardt</surname>
							<given-names>C</given-names>
						</name>
						<name>
							<surname>Guilleme</surname>
							<given-names>M</given-names>
						</name>
						<name>
							<surname>Seeler</surname>
							<given-names>JS</given-names>
						</name>
						<name>
							<surname>Trouche</surname>
							<given-names>D</given-names>
						</name>
						<name>
							<surname>Dejean</surname>
							<given-names>A</given-names>
						</name>
						<name>
							<surname>Yaniv</surname>
							<given-names>M</given-names>
						</name>
					</person-group>
					<article-title>Coordinated methyl and RNA binding is required for heterochromatin localization of mammalian HP1a.</article-title>
					<source>EMBO Rep</source>
					<year>2002</year>
					<volume>3</volume>
					<fpage>975</fpage>
					<lpage>981</lpage>
					<pub-id pub-id-type="pmid">12231507</pub-id>
					<pub-id pub-id-type="doi">10.1093/embo-reports/kvf194</pub-id>
				</citation>
			</ref>
			<ref id="B16">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Grewal</surname>
							<given-names>SI</given-names>
						</name>
						<name>
							<surname>Klar</surname>
							<given-names>AJ</given-names>
						</name>
					</person-group>
					<article-title>Chromosomal inheritance of epigenetic states in fission yeast during mitosis and meiosis.</article-title>
					<source>Cell</source>
					<year>1996</year>
					<volume>86</volume>
					<fpage>95</fpage>
					<lpage>101</lpage>
					<pub-id pub-id-type="pmid">8689692</pub-id>
				</citation>
			</ref>
			<ref id="B17">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Hannon</surname>
							<given-names>GJ</given-names>
						</name>
					</person-group>
					<article-title>RNA interference.</article-title>
					<source>Nature</source>
					<year>2002</year>
					<volume>418</volume>
					<fpage>244</fpage>
					<lpage>251</lpage>
					<pub-id pub-id-type="pmid">12110901</pub-id>
					<pub-id pub-id-type="doi">10.1038/418244a</pub-id>
				</citation>
			</ref>
			<ref id="B18">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Matzke</surname>
							<given-names>M</given-names>
						</name>
						<name>
							<surname>Matzke</surname>
							<given-names>AJ</given-names>
						</name>
						<name>
							<surname>Kooter</surname>
							<given-names>JM</given-names>
						</name>
					</person-group>
					<article-title>RNA: guiding gene silencing.</article-title>
					<source>Science</source>
					<year>2001</year>
					<volume>293</volume>
					<fpage>1080</fpage>
					<lpage>1083</lpage>
					<pub-id pub-id-type="pmid">11498576</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1063051</pub-id>
				</citation>
			</ref>
			<ref id="B19">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Wood</surname>
							<given-names>V</given-names>
						</name>
						<name>
							<surname>Gwilliam</surname>
							<given-names>R</given-names>
						</name>
						<name>
							<surname>Rajandream</surname>
							<given-names>MA</given-names>
						</name>
						<name>
							<surname>Lyne</surname>
							<given-names>M</given-names>
						</name>
						<name>
							<surname>Lyne</surname>
							<given-names>R</given-names>
						</name>
						<name>
							<surname>Stewart</surname>
							<given-names>A</given-names>
						</name>
						<name>
							<surname>Sgouros</surname>
							<given-names>J</given-names>
						</name>
						<name>
							<surname>Peat</surname>
							<given-names>N</given-names>
						</name>
						<name>
							<surname>Hayles</surname>
							<given-names>J</given-names>
						</name>
						<name>
							<surname>Baker</surname>
							<given-names>S</given-names>
						</name>
						<name>
							<surname>Basham</surname>
							<given-names>D</given-names>
						</name>
						<etal/>
					</person-group>
					<article-title>The genome sequence of <italic>Schizosaccharomyces pombe.</italic>
					</article-title>
					<source>Nature</source>
					<year>2002</year>
					<volume>415</volume>
					<fpage>871</fpage>
					<lpage>880</lpage>
					<pub-id pub-id-type="pmid">11859360</pub-id>
					<pub-id pub-id-type="doi">10.1038/nature724</pub-id>
				</citation>
			</ref>
			<ref id="B20">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Reinhart</surname>
							<given-names>BJ</given-names>
						</name>
						<name>
							<surname>Bartel</surname>
							<given-names>DP</given-names>
						</name>
					</person-group>
					<article-title>Small RNAs correspond to centromere heterochromatic repeats.</article-title>
					<source>Science</source>
					<year>2002</year>
					<volume>297</volume>
					<fpage>1831</fpage>
					<pub-id pub-id-type="pmid">12193644</pub-id>
					<pub-id pub-id-type="doi">10.1126/science.1077183</pub-id>
				</citation>
			</ref>
			<ref id="B21">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Grewal</surname>
							<given-names>SI</given-names>
						</name>
						<name>
							<surname>Klar</surname>
							<given-names>AJ</given-names>
						</name>
					</person-group>
					<article-title>A recombinationally repressed region between mat2 and <italic>mat3 </italic>loci shares homology to centromeric repeats and regulates directionality of mating-type switching in fission yeast.</article-title>
					<source>Genetics</source>
					<year>1997</year>
					<volume>146</volume>
					<fpage>1221</fpage>
					<lpage>1238</lpage>
					<pub-id pub-id-type="pmid">9258669</pub-id>
				</citation>
			</ref>
			<ref id="B22">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Ekwall</surname>
							<given-names>K</given-names>
						</name>
						<name>
							<surname>Olsson</surname>
							<given-names>T</given-names>
						</name>
						<name>
							<surname>Turner</surname>
							<given-names>BM</given-names>
						</name>
						<name>
							<surname>Cranston</surname>
							<given-names>G</given-names>
						</name>
						<name>
							<surname>Allshire</surname>
							<given-names>RC</given-names>
						</name>
					</person-group>
					<article-title>Transient inhibition of histone deacetylation alters the structural and functional imprint at fission yeast centromeres.</article-title>
					<source>Cell</source>
					<year>1997</year>
					<volume>91</volume>
					<fpage>1021</fpage>
					<lpage>1032</lpage>
					<pub-id pub-id-type="pmid">9428524</pub-id>
				</citation>
			</ref>
			<ref id="B23">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Ekwall</surname>
							<given-names>K</given-names>
						</name>
						<name>
							<surname>Nimmo</surname>
							<given-names>ER</given-names>
						</name>
						<name>
							<surname>Javerzat</surname>
							<given-names>JP</given-names>
						</name>
						<name>
							<surname>Borgstrom</surname>
							<given-names>B</given-names>
						</name>
						<name>
							<surname>Egel</surname>
							<given-names>R</given-names>
						</name>
						<name>
							<surname>Cranston</surname>
							<given-names>G</given-names>
						</name>
						<name>
							<surname>Allshire</surname>
							<given-names>R</given-names>
						</name>
					</person-group>
					<article-title>Mutations in the fission yeast silencing factors Clr4<sup>+ </sup>and Rik1<sup>+ </sup>disrupt the localisation of the chromo domain protein Swi6p and impair centromere function.</article-title>
					<source>J Cell Sci</source>
					<year>1996</year>
					<volume>109</volume>
					<fpage>2637</fpage>
					<lpage>2648</lpage>
					<pub-id pub-id-type="pmid">8937982</pub-id>
				</citation>
			</ref>
			<ref id="B24">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Partridge</surname>
							<given-names>JF</given-names>
						</name>
						<name>
							<surname>Scott</surname>
							<given-names>KS</given-names>
						</name>
						<name>
							<surname>Bannister</surname>
							<given-names>AJ</given-names>
						</name>
						<name>
							<surname>Kouzarides</surname>
							<given-names>T</given-names>
						</name>
						<name>
							<surname>Allshire</surname>
							<given-names>RC</given-names>
						</name>
					</person-group>
					<article-title>
						<italic>cis</italic>-acting DNA from fission yeast centromeres mediates histone H3 methylation and recruitment of silencing factors and cohesin to an ectopic site.</article-title>
					<source>Curr Biol</source>
					<year>2002</year>
					<volume>12</volume>
					<fpage>1652</fpage>
					<lpage>1660</lpage>
					<pub-id pub-id-type="pmid">12361567</pub-id>
					<pub-id pub-id-type="doi">10.1016/S0960-9822(02)01177-6</pub-id>
				</citation>
			</ref>
			<ref id="B25">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Aravin</surname>
							<given-names>AA</given-names>
						</name>
						<name>
							<surname>Naumova</surname>
							<given-names>NM</given-names>
						</name>
						<name>
							<surname>Tulin</surname>
							<given-names>AV</given-names>
						</name>
						<name>
							<surname>Vagin</surname>
							<given-names>VV</given-names>
						</name>
						<name>
							<surname>Rozovsky</surname>
							<given-names>YM</given-names>
						</name>
						<name>
							<surname>Gvozdev</surname>
							<given-names>VA</given-names>
						</name>
					</person-group>
					<article-title>Double-stranded RNA-mediated silencing of genomic tandem repeats and transposable elements in the <italic>D. melanogaster </italic>germline.</article-title>
					<source>Curr Biol</source>
					<year>2001</year>
					<volume>11</volume>
					<fpage>1017</fpage>
					<lpage>1027</lpage>
					<pub-id pub-id-type="pmid">11470406</pub-id>
					<pub-id pub-id-type="doi">10.1016/S0960-9822(01)00299-8</pub-id>
				</citation>
			</ref>
			<ref id="B26">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Taverna</surname>
							<given-names>SD</given-names>
						</name>
						<name>
							<surname>Coyne</surname>
							<given-names>RS</given-names>
						</name>
						<name>
							<surname>Allis</surname>
							<given-names>CD</given-names>
						</name>
					</person-group>
					<article-title>Methylation of histone H3 at lysine 9 targets programmed DNA elimination in tetrahymena.</article-title>
					<source>Cell</source>
					<year>2002</year>
					<volume>110</volume>
					<fpage>701</fpage>
					<lpage>711</lpage>
					<pub-id pub-id-type="pmid">12297044</pub-id>
				</citation>
			</ref>
			<ref id="B27">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Heard</surname>
							<given-names>E</given-names>
						</name>
						<name>
							<surname>Rougeulle</surname>
							<given-names>C</given-names>
						</name>
						<name>
							<surname>Arnaud</surname>
							<given-names>D</given-names>
						</name>
						<name>
							<surname>Avner</surname>
							<given-names>P</given-names>
						</name>
						<name>
							<surname>Allis</surname>
							<given-names>CD</given-names>
						</name>
						<name>
							<surname>Spector</surname>
							<given-names>DL</given-names>
						</name>
					</person-group>
					<article-title>Methylation of histone H3 at Lys-9 is an early mark on the X chromosome during X inactivation.</article-title>
					<source>Cell</source>
					<year>2001</year>
					<volume>107</volume>
					<fpage>727</fpage>
					<lpage>738</lpage>
					<pub-id pub-id-type="pmid">11747809</pub-id>
				</citation>
			</ref>
			<ref id="B28">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Maison</surname>
							<given-names>C</given-names>
						</name>
						<name>
							<surname>Bailly</surname>
							<given-names>D</given-names>
						</name>
						<name>
							<surname>Peters</surname>
							<given-names>AH</given-names>
						</name>
						<name>
							<surname>Quivy</surname>
							<given-names>JP</given-names>
						</name>
						<name>
							<surname>Roche</surname>
							<given-names>D</given-names>
						</name>
						<name>
							<surname>Taddei</surname>
							<given-names>A</given-names>
						</name>
						<name>
							<surname>Lachner</surname>
							<given-names>M</given-names>
						</name>
						<name>
							<surname>Jenuwein</surname>
							<given-names>T</given-names>
						</name>
						<name>
							<surname>Almouzni</surname>
							<given-names>G</given-names>
						</name>
					</person-group>
					<article-title>Higher-order structure in pericentric heterochromatin involves a distinct pattern of histone modification and an RNA component.</article-title>
					<source>Nat Genet</source>
					<year>2002</year>
					<volume>30</volume>
					<fpage>329</fpage>
					<lpage>334</lpage>
					<pub-id pub-id-type="pmid">11850619</pub-id>
					<pub-id pub-id-type="doi">10.1038/ng843</pub-id>
				</citation>
			</ref>
			<ref id="B29">
				<citation citation-type="journal">
					<person-group person-group-type="author">
						<name>
							<surname>Moazed</surname>
							<given-names>D</given-names>
						</name>
					</person-group>
					<article-title>Common themes in mechanisms of gene silencing.</article-title>
					<source>Mol Cell</source>
					<year>2001</year>
					<volume>8</volume>
					<fpage>489</fpage>
					<lpage>498</lpage>
					<pub-id pub-id-type="pmid">11583612</pub-id>
				</citation>
			</ref>
		</ref-list>
		<sec sec-type="display-objects">
			<title>Figures and Tables</title>
			<fig position="float" id="F1">
				<label>Figure 1</label>
				<caption>
					<p>A model for heterochromatin assembly at the <italic>S. pombe </italic>mating-type locus. RNA transcription, presumably from repeated sequences within the locus, produces dsRNA that is rapidly cleaved by Dicer to form siRNAs. The siRNAs target enzymes to the chromatin so as to methylate histone H3 at lysine 9 - H3K9(Me) on the figure. Histone methylation then recruits the Swi6 protein to the heterochromatin nucleation site. The binding of Swi6 to methylated lysine 9 of histone H3 in this nucleation region then promotes spreading of the silent heterochromatin structure into adjacent sequences by further methylation of H3 at lysine 9 and its association with Swi6, throughout the mating-type locus. Broadly similar mechanisms are likely to work in the centromere. Additional factors are discussed further in the text.</p>
				</caption>
				<graphic xlink:href="gb-2002-3-12-reviews1035-1"/>
			</fig>
		</sec>
	</back>
</article>


